_data/chains/eip155-207.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "VinuChain Network",
+  "chain": "VinuChain",
+  "icon": "vinuchain",
+  "rpc": ["https://vinuchain-rpc.com"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "VinuChain",
+    "symbol": "VС",
+    "decimals": 18
+  },
+  "infoURL": "https://vitainu.org",
+  "shortName": "VC",
+  "chainId": 207,
+  "networkId": 207,
+  "icon": "vitainu",
+  "explorers": [
+    {
+      "name": "VinuScan",
+      "url": "https://vinuscan.com",
+      "icon": "vinuscan",
+      "standard": "EIP3091"
```

does not seem to be compatible

```suggestion
      "standard": "none"
```